worldengine/basic_map_operations.py

```diff
@@ -1,9 +1,12 @@
 import math
-import random
+import numpy
 
 
-def random_point(width, height):
-    return random.randrange(0, width), random.randrange(0, height)
+def random_point(width, height, randint_substitute=None):
```

Comment:
I added this after working on adding a similar feature to the noise module.
Whoever calls this now can make use of its *own* RNG instead of the system-wide one.